src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -26,7 +27,9 @@ attributes:
 
 .Ephemeral Storage
 
-With this kind of storage files are mounted to emptyDir. As the name says, it is initially empty. When a Pod is removed from a node for any reason, the data in the emptyDir is deleted forever.  This means that you lose all your changes if you stop your workspace (or workspace pod restarts for any reason).
+With this kind of storage files are mounted to `emptyDir`. As the name says, it is initially empty. When a Pod is removed from a node for any reason, the data in the `emptyDir` is deleted forever.  This means that you lose all your changes if you stop your workspace (or workspace pod restarts for any reason). Here more about `emptyDir`:link:https://kubernetes.io/docs/concepts/storage/volumes/#emptydir[Volumes:emptyDir]
+
+WARNING: In order to save the changes, one must commit and push to the remote before stopping an ephemeral workspace. Otherwise, all the changes will be lost
```

Comment:
```suggestion
WARNING: In order to save the changes, one must commit and push to the remote before stopping an ephemeral workspace. Otherwise, all the changes will be lost.
```